DoLah/Storage/CalendarStorage.cpp

```diff
@@ -112,6 +112,7 @@ namespace DoLah {
                 }
             }
         }
+        calendar.updateTaskExpiry();
```

Comment:
@bsmmoon  Expiry should not be updated upon loaded. but rather should be computed directly using `AbstractTask::isExpire()`